app/packages/relay/src/mutations/searchSelectFields.ts

```diff
@@ -0,0 +1,9 @@
+import { graphql } from "react-relay";
+
+export default graphql`
+  mutation searchSelectFieldsMutation($metaFilter: JSON = null) {
```

Comment:
Not important right now, but I do think this is better as query field because it is just a search in the end

Reply:
It is in my todo list :)